unlock-app/src/config.js

```diff
@@ -97,7 +97,7 @@ export default function configure(
     )
     blockTime = 10 // in mseconds.
     supportedProviders = ['HTTP']
-    services['storage'] = { host: 'http://127.0.0.1:8080' }
+    services['storage'] = { host: runtimeConfig.locksmithHost }
```

Comment:
Is there a default value for this set in `config.next.js`? I just want to make sure things run fine in dev...